examples/wCCD/src/wCCD.tsx

```diff
@@ -130,6 +136,18 @@ async function updateWCCDBalanceAccount(rpcClient: ConcordiumGRPCClient, account
     return BigInt(leb.decodeULEB128(toBuffer(ReturnValue.toHexString(res.returnValue).slice(4), 'hex'))[0]);
 }
 
+function addWCDToWallet(
+    _accountAddress: string,
+    connection: BrowserWalletConnector,
```

Comment:
Used -> `connection: BrowserWalletConnector`,
Instead of -> `client: WalletApi`,
Because **WalletApi** need to be imported from `@concordium/browser-wallet-api-helpers` to avoid redundant dependency